api/lisk.js

```diff
@@ -155,30 +155,23 @@ module.exports = function (app) {
             },
             sendTransaction : function (cb) {
                 var amount      = app.locals.amountToSend * req.fixedPoint;
-                var transaction = lisk.transaction.createTransaction(address, amount, app.locals.passphrase);
-
-                request({
-                    url : req.lisk + "/peer/transactions",
-                    method : "POST",
-                    json : true,
-                    headers: {
-                        'Content-Type': 'application/json',
-                        'nethash': app.locals.nethash,
-                        'broadhash': app.locals.broadhash,
-                        'os': 'lisk-js-api',
-                        'version': app.locals.liskVersion,
-                        'minVersion': app.locals.liskMinVersion,
-                        'port': app.locals.port
-                    },
-                    body : {
-                        transaction: transaction
-                    }
-                }, function (error, resp, body) {
-                    if (error || resp.statusCode != 200 || !body.success) {
-                        return cb("Failed to send transaction");
-                    } else {
-                        return cb(null, body);
+                var transaction = lisk.default.transaction.transfer(
```

Comment:
probably it will be easier to import it like
var lisk = require('lisk-js').default;

Reply:
also if you are using `const` below, probably better to use const here too